### PR TITLE
#72 - Show all tenders list in all landing pages - Aisha Athman Lali

### PR DESCRIPTION
**Pull request by aishaathmanlali**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/pages/LogInForm.js [hunk 1]
```diff
@@ -25,19 +25,21 @@ function LogInForm() {
...
+					navigate("/admin-dashboard");
+					break;
+				case "buyer":
+					navigate("/buyer-dashboard");
+					break;
+				default:
+					navigate("/bidder-dashboard");
+			}
+
+			navigate("/list-tenders");
```
Comment (soonick) [inline client/src/pages/LogInForm.js]: This is not related to this commit. Create another commit that fixes lint checks.

And please make sure your pre-commit check is working. This should never happen

File: client/src/pages/BuyerDashboard.js [hunk 2]
```diff
@@ -9,6 +9,9 @@ export function BuyerDashboard() {
 				<Link to="/publish-tender">
 					<button>Publish Tender</button>
 				</Link>
+				<Link to="/list-tenders">
+					<button>View Tenders</button>
+				</Link>
```
Comment (soonick) [inline client/src/pages/BuyerDashboard.js]: It should show the list of tenders not a button to the list of tenders

File: client/src/pages/BidderDashboard.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import { Link } from "react-router-dom";
+
+const BidderDashboard = () => {
+	return (
+		<main>
+			<h1>Bidder Dashboard</h1>
+			<Link to="/list-tenders">
+				<button>View Tenders</button>
+			</Link>
+		</main>
```
Comment (soonick) [inline client/src/pages/BidderDashboard.js]: It should show the list of tenders not a button to the list of tenders

File: client/src/pages/BidderDashboard.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const BidderDashboard = () => {
+	return (
+		<main>
+			<h1>Bidder Dashboard</h1>
```
Comment (soonick) [inline client/src/pages/BidderDashboard.js]:
```suggestion
			<h1>Tenders</h1>
```

File: client/src/App.js [hunk 5]
```diff
@@ -22,6 +23,7 @@ const App = () => (
 			<Route path="/signup" element={<SignUp />} />
 			<Route path="/admin-dashboard" element={<AdminDashboard />} />
```
Comment (soonick) [inline client/src/App.js:24]: Add the list of thenders to AdminDashboard too

**Review by soonick — CHANGES_REQUESTED** [review]

File: client/src/pages/BidderDashboard.js (hunk 3 above)
Comment (aishaathmanlali) [reply client/src/pages/BidderDashboard.js]: Okay will remove the button

File: client/src/pages/AdminDashboard.js [hunk 6]
```diff
@@ -1,13 +1,11 @@
 import React from "react";
-import { Link } from "react-router-dom";
+import TendersList from "../TenderList";
 
 const AdminDashboard = () => {
 	return (
 		<main>
 			<h1>Admin Dashboard</h1>
```
Comment (soonick) [inline client/src/pages/AdminDashboard.js]: Tenders

File: client/src/pages/BuyerDashboard.js [hunk 7]
```diff
@@ -9,6 +10,7 @@ export function BuyerDashboard() {
 				<Link to="/publish-tender">
 					<button>Publish Tender</button>
 				</Link>
```
Comment (soonick) [inline client/src/pages/BuyerDashboard.js]: Remove all buttons and leave only `tenders` as title

```suggestion
				<h1>Tenders</h1>
```

**Review by soonick — CHANGES_REQUESTED** [review]

File: client/src/pages/BuyerDashboard.js (hunk 7 above)
Comment (aishaathmanlali) [reply client/src/pages/BuyerDashboard.js]: what about the publish button? do i remove it from the dashboard?

File: client/src/pages/LogInForm.js [hunk 8]
```diff
@@ -38,6 +38,8 @@ function LogInForm() {
 				default:
 					navigate("/bidder-dashboard");
 			}
+
+			navigate("/list-tenders");
```
Comment (soonick) [inline client/src/pages/LogInForm.js]: What is this?

File: client/src/pages/AdminDashboard.js (hunk 6 above)
Comment (soonick) [reply client/src/pages/AdminDashboard.js]: @aishaathmanlali . You didn't address this comment

File: client/src/pages/BuyerDashboard.js (hunk 7 above)
Comment (soonick) [reply client/src/pages/BuyerDashboard.js]: Remove the button and only leave the tenders title

**Review by soonick — CHANGES_REQUESTED** [review]

**Review by soonick — APPROVED** [review]
